rollup.config.js

```diff
@@ -15,9 +15,12 @@ export default {
   plugins: [
     nodeResolve(),
     json(),
-    typescript(),
+    typescript({
+      clean: true,
+    }),
     babel({
       exclude: "node_modules/**",
+      babelHelpers: "bundled",
```

Comment:
This was the previous behavior; specifying it explicitly prevents a build-time warning.